.rubocop.yml

```diff
@@ -0,0 +1,128 @@
+inherit_from:
+  - .rubocop_todo.yml
+
+AllCops:
+  TargetRubyVersion: 1.9
```

Comment:
Hey.. what ruby version are you using? I plan on supporting only ruby 2 or above. Can you please update your ruby? Let me know if you need help.